v2/contribute/decisions/session/0030-get-session-from-string.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: "0030"
+title: Add a new verifyAccessToken function that takes the token as a string
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Add a new verifyAccessToken function that takes the token as a string
+
+<DecisionHeader status="proposed" lastUpdate="2022-12-13" created="2022-12-13" deciders={["rishabhpoddar", "porcellus"]} proposedBy={["porcellus"]} />
+
+## Context and Problem Statement
+
+In some cases, the BE may need to validate access tokens based on the string only, with no access to the request/response objects.
+
+## Considered Options
+
+* Recommend using a jwt verification lib
+* Add a new `verifyAccessToken` function that takes the token as a string
+
```

Comment:
Another option: Ask user to mock the req / res object passed to getSession and read tokens from that